frontend/src/components/OrganizationForm/OrganizationForm.tsx

```diff
@@ -8,7 +8,8 @@ import {
   DialogActions,
   Switch,
   Button,
-  FormControlLabel
+  FormControlLabel,
+  ValidatorForm
```

Comment:
Is this library you intended on using?  It's not part of material-ui but is part of react-material-ui-form-validator.

Reply:
I removed the import as it was leftover from previous solutions I was testing. PR is ready for review.